### DISCUSSION TITLE
Project plan

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-taxsim#11] [General]
@noman404 goals by week:

October 4:
* Review taxsim interface from https://taxsim.nber.org/taxsim35/
* Consult https://github.com/PolicyEngine/taxsim (private repo)
* Mapping at least a couple more input variables

October 11:
* Mapping all inputs
* Expand outputs

October 18:
* Complete outputs
* Begin validation
* Begin documentation

October 25:
* Complete validation and documentation

**Comment by feenberg** [comment]
I see many issues I have raised are closed with comments suggesting the code has been updated, but when I do a "git pull" I don't get the updated code, but am told I am up to date.  Is there a way for me to pull the updated code?
